src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -81,100 +81,136 @@ function ExamSettings(props) {
+          </Form.Text>
+        </Form.Group>
+
+        {/* ALLOW OPTING OUT OF PROCTORED EXAMS */}
+        <fieldset aria-describedby="allowOptingOutHelpText">
+          <Form.Group controlId="formAllowingOptingOut">
+            <Form.Label as="legend">Allow Opting Out of Proctored Exams</Form.Label>
+            <Form.Check
+              type="radio"
+              id="allowOptingOutYes"
+              name="allowOptingOut"
+              label="Yes"
+              inline
+              checked={allowOptingOut}
+              onChange={() => onAllowOptingOutChange(true)}
+            />
+            <Form.Check
+              type="radio"
+              id="allowOptingOutNo"
+              name="allowOptingOut"
+              label="No"
+              inline
+              checked={!allowOptingOut}
+              onChange={() => onAllowOptingOutChange(false)}
+            />
+            <Form.Text id="allowOptingOutHelpText">
+              If this value is &quot;Yes&quot;, learners can choose to take proctored exams without proctoring.
+              If this value is &quot;No&quot;, all learners must take the exam with proctoring.
+              This setting only applies if proctored exams are enabled for the course.
+            </Form.Text>
+          </Form.Group>
+        </fieldset>
+
+        {/* PROCTORING PROVIDER */}
+        <Form.Group controlId="formProctoringProvider">
+          <Form.Label>Proctoring Provider</Form.Label>
+          <Form.Control
+            as="select"
             value={proctoringProvider}
             onChange={onProctoringProviderChange}
-          />
-        </ValidationFormGroup>
-        <ValidationFormGroup
-          for="proctortrackEscalationEmail"
-          helpText="Required if 'proctortrack' is selected as your proctoring provider. Enter an email address to be contacted by the support team whenever there are escalations (e.g. appeals, delayed reviews, etc.)."
-        >
-          <label htmlFor="proctortrackEscalationEmail">Proctortrack Escalation Email</label>
-          <Input
-            type="text"
-            id="proctortrackEscalationEmail"
-            name="proctortrack_escalation_email"
-            value={proctortrackEscalationEmail}
+          >
+            {getProctoringProviderOptions(availableProctoringProviders)}
+          </Form.Control>
+          <Form.Text>{cannotEditProctoringProvider() ? ('Proctoring provider cannot be modified after course start date.') : ('Select the proctoring provider you want to use for this course run.')}</Form.Text>
+        </Form.Group>
+
+        {/* PROCTORTRACK ESCALATION EMAIL */}
+        <Form.Group controlId="formProctortrackEscalationEmail">
+          <Form.Label>Proctortrack Escalation Email</Form.Label>
+          <Form.Control
+            type="email"
             onChange={onProctortrackEscalationEmailChange}
+            value={proctortrackEscalationEmail}
+            isInvalid={!!proctortrackEscalationEmailError}
```

Comment:
This adds some additional validation to the actual field. This improves accessibility.

Reply:
Thank you for adding this!